src/app/routes/cpsAsset/getInitialData/addRecommendationsBlock/index.js

```diff
@@ -0,0 +1,46 @@
+import path from 'ramda/src/path';
+import deepClone from 'ramda/src/clone';
+import splitAt from 'ramda/src/splitAt';
+import { getNthCpsParagraphIndex } from '../helpers';
+
+const insertRecommendationsBlock = (recommendationBlock, blocks) => {
+  // get the index of the 5th paragraph
```

Comment:
This is a small thing, and perhaps my preference, but comments like this can detract from what is already clean and readable code. 👍 I would much rather see them explaining things that are less obvious. I found this [section on comments](https://javascript.info/comments) really useful.

Reply:
Thanks @simonsinclair for the resource.